### DISCUSSION TITLE
Month view dot size

### DISCUSSION THREAD

**Discussion by bitdelve** [kvyatkovskys/KVKCalendar#201] [General]
How would I go about changing the dot size for events on the month view?

**Comment by kvyatkovskys** [comment]
You mean, the dot of date in month cell 🤔

**Reply by kvyatkovskys** [reply]
If yes, you can't 🙂

**Reply by bitdelve** [reply]
I mean the dot that indicated events in the cell. Like if there are 2 events on that day then there are two dots.

**Reply by kvyatkovskys** [reply]
Sorry for the long answer 😕
I think no. Need to add an additional functionality, a bit.
